Gemfile

```diff
@@ -177,7 +177,7 @@ gem 'themes_on_rails', '>= 0.3.1', github: 'lowjoel/themes_on_rails',
 
 # Forms made easy for Rails
 gem 'simple_form'
-gem 'simple_form-bootstrap', '>= 1.3.0'
+gem 'simple_form-bootstrap', github: 'allenwq/simple_form-bootstrap'
```

Comment:
@allenwq joel merged the PR already, so we don't need to use this fork any more?